### DISCUSSION TITLE
Add information on CSP policy / unsafe-inline

### DISCUSSION THREAD

**Discussion by unuseless** [adamghill/unsuckjs.com#28] [Ideas]
Some libraries use inline <script> tags or inline event handlers.  These libraries don't work with a content security policy (CSP) without 'unsafe-inline', which is a security risk.
It would be great if the table allows adding that information as it becomes available.
As an example, petite-vue unfortunately won't work with a strict CSP.

**Comment by adamghill** [comment]
If you want to make a PR adding an attribute for everything in [libraries.json](https://github.com/adamghill/unsuckjs.com/blob/main/data/libraries.json) I will write the code to display it.
